Lib/defcon/objects/layoutEngine.py

```diff
@@ -38,15 +38,15 @@ def _layoutEngineOTLTablesRepresentationFactory(layoutEngine):
             addOpenTypeFeatures(feaPath, otf)
```

Comment:
in the current fonttools.feaLib, `addOpenTypeFeatures` accepts both paths and file objects. There's also `addOpenTypeFeaturesFromString` which I think is what you want to use here.